components/login/login.css

```diff
@@ -0,0 +1,45 @@
+.nav-container{
+    z-index: 3;
+}
+.nav-title{
+    font-size: 1.3rem;
+  }
+  .nav-icon{
+    font-size: 1.3rem;
+  }
+.main-content{
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+    text-align: center;
+    gap: 1rem;
+    padding: 1.75rem 0;
+}
+ .description{
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    padding: 2rem;
+} 
+
+.auth-form{
+    display: flex;
+    align-items: center;
+    justify-content: center;
+
+}
+
+.input{
+    border-radius: 20px;
+}
+
+.auth-text{
```

Comment:
suggestion - Try to use generic names for such classes make them for utility purpose

Reply:
Sure I'll keep it in mind from now on